dist/obs-bundled-gems.spec

```diff
@@ -51,6 +51,8 @@ This package bundles all the gems required by the Open Build Service
 to make it easier to deploy the obs-server package.
 
 %define rake_version 12.3.1
+%define passenger_version 5.3.7
```

Comment:
I don't think there is a problem with passenger - as we don't bundle it. rack and rake are special in the sense that we have them as package for passenger *and* in the bundle.

And I would prefer if you checked the bundled rack version in line 106 too

Reply:
Makes sense, updated!